tests/DwitTech.AccountService.Core.Tests/Utilities/RandomUtilTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DwitTech.AccountService.Core.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DwitTech.AccountService.Core.Tests.Utilities
+{
+    public class RandomUtilTests
+    {
+        [Theory]
+        [InlineData("op1wUje9Y1SVY0qWAUS4")]
+        public void GenerateUniqueCode_Returns_Unique_twenty_Character_Alphanum_string(string expected)
```

Comment:
Does this tests pass? If it passes, its suspicious. If it doesn't, why are you pushing a failing test?

Reply:
It's supposed to fail because all the results should be different. What I did was generate one value, use it as my expected, and compare it with ten other values generated to see if it'll generate the same value. Am I only supposed to push tests that pass?

Reply:
Yes. Push only passed tests. I see your point then you should generate 1 as expected and generate 10 more as actuals then Assert that they are not equal.
This test may become flaky but we can make do with it for now.

Reply:
Ok. Will update the test